wgpu/src/backend/web.rs

```diff
@@ -1255,7 +1252,6 @@ impl crate::Context for Context {
                             }
                         };
                         let mut storage_texture = web_sys::GpuStorageTextureBindingLayout::new(
-                            mapped_access,
```

Comment:
[`access`](https://rustwasm.github.io/wasm-bindgen/api/web_sys/struct.GpuStorageTextureBindingLayout.html#method.access) should still be set even if there's only one variant at the moment (this might change again in the short- to mid-term)

Reply:
I've added the call to `access` with the `mapped_access`.